pkg/frontend/metrics.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/recover"
+)
+
+//emitMetrics sends a heartbeat metric every 60 seconds
+func (f *frontend) emitMetrics(stop <-chan struct{}) {
+	defer recover.Panic(f.baseLog)
+	t := time.NewTicker(time.Minute)
+	defer t.Stop()
+	f.baseLog.Print("starting heartbeat")
+
+	for range t.C {
+		select {
+		case <-stop:
+			return
+		default:
+			if f.CheckReady() {
+				f.m.EmitGauge("frontend.healthz.ready", 1, map[string]string{
+					"status": "ready",
+				})
+			} else {
+				f.m.EmitGauge("frontend.healthz.ready", 1, map[string]string{
+					"status": "notready",
+				})
```

Comment:
Are we better off removing the status dimension and solely emitting if ready is true?  Does it reduce our chances of misconfiguring the Geneva side?

Reply:
It reduces the chances, but if we want to alert on RPs in "not ready" for some reason we should have a separate metric for that.

Reply:
In the end after chatting with @mjudeikis the "notready" heartbeat has very little utility. We'll have to check the logs if we don't see a proper heartbeat coming up anyway.

I removed the whole else clause.